### ISSUE TITLE
LineChart bug when data full of "0"

### ISSUE THREAD

**Issue by SimonTernoir** [chartist-js/chartist#40]
Hi! Thanks for your great job on this project!

I've tested lineChart with different series of data and I'd like to report a bug when a data array is full of "0". 

Lots of errors are printed in console like:
`Error: Invalid value for <line> attribute y2="NaN"`
And finally, the console prints the following error:
`Error: Invalid value for <path> attribute d="M20,NaN L 41.71875,NaNL 63.4375,NaNL 85.15625,NaNL 106.875,NaNL 128.59375,NaNL 150.3125,NaNL 172.03125,NaNL 193.75,NaNL 215.46875,NaNL 237.1875,NaNL 258.90625,NaNL 280.625,NaNL 302.34375,NaNL 324.0625,NaNL 345.78125,NaNL 367.5,NaNL 389.21875,NaNL 410.9375,NaNL 432.65625,NaNL 454.375,NaNL 476.09375,NaNL 497.8125,NaNL 519.53125,NaNL 541.25,NaNL 562.96875,NaNL 584.6875,NaNL 606.40625,NaNL 628.125,NaNL 649.84375,NaNL 671.5625,NaNL 693.28125,NaN"`

The problem seems coming from the function `Chartist.getBounds` which returns NaN values but I won't have more time for investigating shortly. 

-Simon

**Comment by gionkunz** [comment]
Hi Simon! Thanks for your bug report. Can you give me a an exact data example that causes this issue? My guess is that the catmoll rom function will fail with too many weird values. It would be helpful to have an example though, but I'll try to reproduce myself.

Cheers
Gion

**Comment by paulswail** [comment]
I am getting the same error. Here's the code to reproduce:

```
var chartData = {
            labels: [],
            series: [ [0,0,0,0,0] ]
        };
Chartist.Line('#myChart', chartData, { showLabel: false, showPoint: false});
```

**Comment by gionkunz** [comment]
Hi thanks for the example. I've created this: http://jsbin.com/muboju/1/edit

It looks like there is an issue if all values from all series have the same value, no matter if smoothed or not.

I'll mark this issue as bug and fix it ASAP.

Thanks for the report!

Cheers
Gion

**Comment by SimonTernoir** [comment]
Thanks Paul you've been faster than me :)

**Comment by gionkunz** [comment]
This is fixed and Chartist v0.1.14 is just pushed
